lib/lib.es6.d.ts

```diff
@@ -5556,7 +5556,7 @@ interface Symbol {
     toString(): string;
 
     /** Returns the primitive value of the specified object. */
-    valueOf(): Object;
```

Comment:
can you revert these changes. all you need is the change to `src/lib/es2015.symbol.d.ts`.